chia-protocol/Cargo.toml

```diff
@@ -12,15 +12,16 @@ repository = "https://github.com/Chia-Network/chia_rs/chia-protocol/"
 py-bindings = ["dep:pyo3", "dep:chia_py_streamable_macro", "chia-traits/py-bindings"]
 
 [dependencies]
-pyo3 = { version = "0.19.0", features = ["multiple-pymethods"], optional = true }
+pyo3 = { version = "0.19.0", features = ["multiple-pymethods", "num-bigint"], optional = true }
```

Comment:
the `num-bigint` feature is to enable support PyO3 support for converting python `int` to `BigInt` types in rust.